cryptostore/data/gd.py

```diff
@@ -0,0 +1,153 @@
+'''
+Copyright (C) 2018-2020  Bryant Moscon - bmoscon@gmail.com
```

Comment:
You can just leave the copyright off if you want, or you can add something about how you are the author. Or you can put on a public domain header with your name on it.